docs/release-notes/flink-1.10.md

```diff
@@ -376,3 +406,8 @@ external thread). The methods `MailboxExecutor#yield()` or
 control to other actions temporarily, e.g., if the current operator is
 blocked. The `MailboxExecutor` can be accessed by using
 `YieldingOperatorFactory` (see `AsyncWaitOperator` for an example usage).
+
+#### Deprecation of OptionsFactory and ConfigurableOptionsFactory interfaces ([FLINK-14926](https://issues.apache.org/jira/browse/FLINK-14926))
+Interfaces `OptionsFactory` and `ConfigurableOptionsFactory` have been
+deprecated in favor of `RocksDBOptionsFactory` and
+`ConfigurableRocksDBOptionsFactory`, respectively.
```

Comment:
Also it is worth mentioning removal of the already deprecated `-yst` flag mentioned here:

https://issues.apache.org/jira/browse/FLINK-14957

Reply:
done